### BIG differences, this is diff of wrong files.
### CLUS "System Class PATHNAME"
### and CLHS topic selection http://clhs.lisp.se/Body/a_pn.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:pathname.txt	2017-04-27 21:46:47.000000000 +0100
+++ saved-pages-clus/cl:types:pathname.txt	2017-04-27 21:46:47.000000000 +0100
@@ -1,4 +1,16 @@
-PATHNAME
-Please select which reference to PATHNAME you intended
+System Class PATHNAME

-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Class Precedence List
+pathname, t
+Description
+A pathname is a structured object which represents a filename.
+There are two kinds of pathnames - physical pathnames and logical pathnames.
\ No newline at end of file
